### PR TITLE
[scheduler] Fix scheduler templating

### PR DESCRIPTION
**Pull request by danking**

### CODE REVIEW

File: scheduler/Dockerfile [hunk 1]
```diff
@@ -2,6 +2,7 @@ FROM {{ service_base_image.image }}
 
 COPY scheduler/setup.py /scheduler/
 COPY scheduler/scheduler/ /scheduler/scheduler/
+COPY templates /scheduler/templates
```
Comment (cseed) [inline scheduler/Dockerfile]: The right way to do this is with a MANIFEST.in.

Comment (danking) [reply scheduler/Dockerfile]: The templates directory isn't inside the python package. I tried moving it into the package a la batch, but then Jinja could not find it. Is there something more I need to do?

Comment (cseed) [reply scheduler/Dockerfile]: You should call web_common.setup_aiohttp_jinja2.  That will set up the jinja2 search paths based on a python module (and pull in the web_common stuff which we'll want to switch to eventually).  See e.g. batch/front_end.

**Comment by danking** [comment]
I'll come back to this when it's blocking me.

**Pull request closed without merging**